mrp_sale_info/models/mrp_production.py

```diff
@@ -1,21 +1,38 @@
 # -*- coding: utf-8 -*-
 # © 2016 Antiun Ingenieria S.L. - Javier Iniesta
+# Copyright 2018 Binovo IT Human Project SL
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import models, fields
+from openerp import models, fields, api
 
 
 class MrpProduction(models.Model):
     _inherit = "mrp.production"
 
+    sale_line_id = fields.Many2one(
+        comodel_name='sale.order.line', string='Sale Line',
+        compute='_compute_sale_line_id')
     sale_id = fields.Many2one(
         comodel_name='sale.order', string='Sale order',
         readonly=True, store=True,
-        related='move_prod_id.procurement_id.sale_line_id.order_id')
-    sale_line_id = fields.Many2one(
-        comodel_name='sale.order.line', string='Sale Line',
-        related='move_prod_id.procurement_id.sale_line_id')
-    partner_id = fields.Many2one(related='sale_id.partner_id',
-                                 string='Customer', store=True)
-    commitment_date = fields.Datetime(related='sale_id.commitment_date',
-                                      string='Commitment Date', store=True)
+        related='sale_line_id.order_id')
+    partner_id = fields.Many2one(
+        comodel_name='res.partner', string='Customer',
+        store=True,
+        related='sale_id.partner_id')
+    commitment_date = fields.Datetime(
+        related='sale_id.commitment_date',
+        string='Commitment Date', store=True)
+
+    @api.multi
+    def _compute_sale_line_id(self):
+        def get_parent_move(move_obj):
```

Comment:
usually move_obj will be used to refer to self.env['stock.move'] so change it to only move